pattern-library/sass/patterns/_forms.scss

```diff
@@ -257,6 +257,14 @@ $input-width: (
             border-color: palette(error, text);
         }
     }
+
+    // Input fields immediately followed by a button should eliminate the spacing between them
```

Comment:
What's your plan with this approach? I would prefer it to removing padding before all buttons that happen to be next to an input. Did it not work out?